Keybinding-Issues.md

```diff
@@ -208,8 +208,8 @@ For example, the `Toggle Line Comment` action is bound to <kbd>Ctrl+0xBF</kbd>:
 
 | Kb Layout | Scan Code | Virtual Key | Produced Character |
 |---|---|---|---|
-| US | `[Slash]` | `0xBF` | `/`
-| GER | `[Backslash]` | `0xBF` | `#`
+| US | `[Slash]` | <kbd>0xBF</kbd> | `/`

```

Comment:
This one might be best kept with backticks?

Reply:
I tried to be consistent with marking up references to keys with `<kbd>` tags and keeping backticks for keycodes etc., but I suppose that a) this is a virtual key and b) it may make the table read slightly better, so I shall make it so.